lib/voteman.py

```diff
@@ -0,0 +1,195 @@
+
+    @staticmethod
+    async def tweet_body_parser(tweet_body_message: discord.message, respond_ch: discord.channel, bill_author_id: int):
+        """
+        Parameters
+        ----------
+        tweet_body_message: discord.message
+            ツイート本文のメッセージのチャンネル
+        respond_ch: discord.channel
+            voteが始められたチャンネルのオブジェクト
+        bill_author_id: int
+            法案の提出者のID
+        Returns
+        ----------
+        parsed_text: str
+            実際に加工されたあとのテキスト
+        """
+        tmp = tweet_body_message.content
+
+        # コマンドのprefixとその後の空白(全角/半角)を消す
+        if "!tw　" in tmp:
+            tmp = tmp.replace("!tw　", "", 1)
+        if "!tw " in tmp:
+            tmp = tmp.replace("!tw ", "", 1)
+
+        # Discord絵文字が入っている場合に警告して、更に取り除く
+        if len(re.findall(ORIGINAL_EMOJI_REGEX, tmp)) >= 1:
+            s = "<@!{}>\nDiscord emoji が入っています！\n絵文字は無視されます！".format(bill_author_id)
+            await respond_ch.send(s)
+            tmp = re.sub(ORIGINAL_EMOJI_REGEX, "", tmp)
+
+        parsed_text = tmp
+
+        return parsed_text
+
+    async def status_changer(self, reaction_member: discord.member, emoji_type: str, status: str):
+        """
+        ステータスを変更するための関数
+        Parameters
+        ----------
+        reaction_member: discord.member
+            リアクションを行ったメンバーのオブジェクト
+        emoji_type: str
+            その絵文字がACなのかWAなのかが格納されている
+        status: str
+            Reactionがadd,rem,clrのいずれかが格納される
+        """
+        if status == "add":
+            member_id = reaction_member.id
+            # つけられた絵文字のほうじゃない絵文字が格納される
+            complement_emoji = "AC" if emoji_type == "WA" else "WA"
+            if member_id in self.vote_result[complement_emoji]:
+                await self.vote_target_msg.remove_reaction(MessageManager.EMOJI_DICT[complement_emoji], reaction_member)
+            self.vote_result[emoji_type].add(member_id)
+
+        if status == "rem":
+            member_id = reaction_member.id
+            self.vote_result[emoji_type].remove(member_id)
+
+        if status == "clr":
+            await MessageManager.VOTE_CH.send("クリアすんなカス！！！！㊙すぞ！！！")
+            self.vote_result = {"AC": set(), "WA": set()}
+
+        print("現在の投票状況：")
+        print(self.vote_result)
+
+    @staticmethod
+    async def status_changer_wrapper(status: str, message_id: int, member: discord.member = None, emoji_type: str = None):
+        """
+        ステータスを変更するための関数
+        Parameters
+        ----------
+        message_id: int
+            ステータスが変更されたメッセージのID
+        member: discord.member
+            リアクションを行ったメンバーのオブジェクト
+        emoji_type: str
+            その絵文字がACなのかWAなのかが格納されている
+        status: str
+            Reactionがadd,rem,clrのいずれかが格納される
+
+        """
+        o = MessageManager.MESSAGE_INSTANCES[message_id]
+        await o.status_changer(member, emoji_type, status)
+
+    @staticmethod
+    def static_init(twitter_vote_ch_obj: discord.TextChannel, citizen_list, emoji_dict: dict):
```

Comment:
これを、こうじゃ
```suggestion
    def static_init(twitter_vote_ch_obj: discord.TextChannel, citizen_list: list[discord.Member], emoji_dict: dict[str:discord.Emoji]):
```
一行が長くなってしまうのでこれでもいいかもしれません：
```suggestion
    def static_init(
            twitter_vote_ch_obj: discord.TextChannel, 
            citizen_list: list[discord.Member], 
            emoji_dict: dict[str:discord.Emoji]
    ):
```